src/main/java/edu/whimc/portals/Destination.java

```diff
@@ -7,28 +7,63 @@
 import org.bukkit.Location;
 import org.bukkit.entity.Player;
 
+/**
+ * The class to store Destination data.
+ */
 public class Destination {
-
+    /* Keyword for no destination. */
```

Comment:
```suggestion
    /** Keyword for no destination. */
```
I believe one-liner javadocs like these should start with a double asterisk